samples/HttpRequestFeaturesv2/HttpRequestFeaturesExample.sln

```diff
@@ -0,0 +1,25 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio Version 16
+VisualStudioVersion = 16.0.30011.19
+MinimumVisualStudioVersion = 10.0.40219.1
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "HttpRequestFeaturesExample", "HttpRequestFeaturesExample.csproj", "{46FF071B-A231-4C4F-9401-F76C9ADCBCDC}"
+EndProject
```

Comment:
I don't see the removal of HttpRequestFeaturesExample as we discussed.

Reply:
I was going to remove that and rename this in a separate pr, but I can do that here instead

Reply:
I would do it here to keep the second PR with the rename clean.